modules/environment/default.nix

```diff
@@ -46,7 +46,8 @@ in
     environment.postBuild = mkOption {
       type = types.lines;
       default = "";
-      description = "Commands to execute when building the global environment.";
+      visible = false;
+      description = "Deprecated alias of {option}`environment.extraSetup`";
```

Comment:
What's the reason behind deprecating `environment.postBuild`?

Reply:
NixOS uses `environment.extraSetup` for that, and having two options doing the same thing is error-prone, so I thought I should introduce the NixOS option and make the old one warn on use.